apps/nextjs/package.json

```
@@ -9,7 +9,7 @@
     "test:e2e:production": "NEXT_PUBLIC_ANALYTICS_MODE=production pnpm --filter nextjs... build && playwright test production"
   },
   "dependencies": {
-    "@vercel/analytics": "workspace:^0.1.5",
+    "@vercel/analytics": "workspace:../../packages/web",
```

? Why's this changed?

---

we should always test against latest package? tests where failing and after I changed that it worked again

---

```suggestion
    "@vercel/analytics": "workspace:*",
```

I think i see the issue - using `^0.1.5` means it'll only use the local version while semver matches `^0.1.5`. Bumping the package could break that. 

Instead, we want this test app to _always_ use the workspace package. 
So we should have `workspace:*`.